app/v3/controllers/SubmitSelfEmploymentBsasController.scala

```diff
@@ -134,10 +134,11 @@ class SubmitSelfEmploymentBsasController @Inject()(val authService: EnrolmentsAu
       case NotFoundError   => NotFound(Json.toJson(errorWrapper))
       case DownstreamError => InternalServerError(Json.toJson(errorWrapper))
     }
+
```

Comment:
The `FormatAdjustmentValueError` above needs updating to `ValueFormatError`  on line 128